src/App.tsx

```diff
@@ -1,18 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const total = items.length;
+  const [perPage, setPerPage] = useState(5);
```

Comment:
What 5 mean? Maybe you need to create an array for possible perPage values?

Reply:
I am fine like this, as there may be infinite values possible. 5 is the default, i will create a variable for it so that it gets clearer